packages/server/database/migrations/20210224121516-mergeDuplicateUsers.ts

```diff
@@ -106,15 +106,63 @@ export const up = async function(r: R) {
   }
 
   try {
-    const affectedEmails = (await r
-      .table('User')
-      .group('email')
-      .count()
-      .ungroup()
-      .filter((row) => row('reduction').gt(1))('group')
-      .filter((row) => row.ne('DELETED'))
-      .run({arrayLimit: 200000})) as string[]
-    
+    // hard coded bc we don't want to risk running the agg query in prod
+    // may need to get a fresh list before running backfill in prod
+    const affectedEmails = [
```

Comment:
we can't have any PII in our source code. can you do a hard reset on this commit?

consider storing this list in a DB table or a JSON file that is imported during CI

Reply:
yes. force pushed a rebase. will get the emails from a DB table